README.md

```diff
@@ -34,11 +34,14 @@ The project is currently in a Production/Stable stage, which means that the curr
 
 ### Requirements
 
-Python 3+
+Python >= 3.0
+stix2 >= 2.1.0
+taxii2-client >= 2.2.2
+six >= 1.15.0
```

Comment:
taxii2-client >= 2.3.0
six >= 1.16.0

Reply:
@Cyb3rWard0g anything else you would like to add and change in these files ?

Reply:
I believe other than that, it looks good! Thank you again for contributing to the project :)